statefun-sdk-go/v3/pkg/statefun/egress.go

```diff
@@ -78,7 +78,7 @@ func (k KafkaEgressBuilder) toEgressMessage() (*protocol.FromFunction_EgressMess
 	} else {
 		switch value := k.Value.(type) {
 		case string:
-			_ = StringType.Serialize(&buffer, value)
+			buffer.WriteString(value)
```

Comment:
I think that there is an encoding issues here, as I understand strings in Go are not necessarily `utf-8` encoded.
(constants are utf8, while at runtime they are not necessarily)
Do you think that this is an issue worth addressing?

Perhaps at least we need to add a comment in the method documentation? to use to correct string encoding to match the expected encoding in the Kafka topic. 
This is a slight difference from the Python and Java SDKs as there we also auto deduce strings to be utf-8 strings.

Reply:
You are correct, I added an explicit utf-8 encoding step.